internal/customizations/customizations.go

```diff
@@ -0,0 +1,101 @@
+package customizations
+
+import (
+	"github.com/project-flotta/osbuild-operator/api/v1alpha1"
+)
+
+var void struct{}
+
+func MergeCustomizations(templateCustomizations, configCustomizations *v1alpha1.Customizations) *v1alpha1.Customizations {
+	var customizations *v1alpha1.Customizations
+	if configCustomizations != nil {
```

Comment:
I think this block (including the `if`) should be the `else` block of the next `if` because you override `customizations` anyhow

Reply:
Yeah, that will work too.